### ISSUE TITLE
develop -> command is inconsistent with rest of BUILD.yaml config

### ISSUE THREAD

**Issue by fbarl** [tmrowco/brick#41]
Looking through different brick `BUILD.yaml` files, I noticed that in some steps `command` is used while in some others there is a list of `commands`. I assumed the two are actually interchangeable and that both can be used across `build`, `test`, `develop` etc, only to end up trying to figure why `test` -> `command` config is always passing the tests - turns out the `commands` list being empty means no command have been run so it always passes.

Looking at the code, it seems that only `develop` uses a singular `command` while all the other steps require `commands`. I have a few suggestions how we could improve the behaviour:

1. Switch to `commands` syntax in `develop` and completely drop the `command` option
2. Support both `command` and `commands` options across all steps
3. Keep things as they are but document the examples better to make it clear `develop` steps are an exception in the sense that only there exactly one command is expected

See https://github.com/tmrowco/tmrow/issues/3236 for more context.

**Comment by corradio** [comment]
Good catch! I think the best solution is to support both. That would make sure the tool stays intuitive!
